lib/cli/src/Cardano/CLI.hs

```diff
@@ -716,6 +717,13 @@ cmdMnemonic = command "mnemonic" $ info (helper <*> cmds) $ mempty
         <> cmdMnemonicGenerate
         <> cmdMnemonicRewardCredentials
 
+cmdMnemonicByron :: Mod CommandFields (IO ())
+cmdMnemonicByron = command "mnemonic" $ info (helper <*> cmds) $ mempty
+    <> progDesc "Manage mnemonic phrases."
```

Comment:
Slight duplication but `cmdMnemonic` and `cmdMnemonicByron` are right next to each other (difficult to forget about) so seems just fine 👍